subsys/bluetooth/host/classic/Kconfig

```diff
@@ -26,6 +27,86 @@ config BT_MAX_SCO_CONN
 	  Maximum number of simultaneous Bluetooth synchronous connections
 	  supported. The minimum (and default) number is 1.
 
+config BT_L2CAP_RET
+	bool "Bluetooth L2CAP retransmission mode [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables Bluetooth L2CAP retransmission mode
+
+config BT_L2CAP_FC
+	bool "Bluetooth L2CAP flow control mode [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables Bluetooth L2CAP flow control mode
+
+config BT_L2CAP_ENH_RET
+	bool "Bluetooth L2CAP enhance retransmission [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables Bluetooth L2CAP enhance retransmission
+
+config BT_L2CAP_STREAM
+	bool "Bluetooth L2CAP streaming mode [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables Bluetooth L2CAP streaming mode
+
+config BT_L2CAP_FCS
+	bool "Bluetooth L2CAP FCS Option [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	depends on BT_L2CAP_STREAM || BT_L2CAP_ENH_RET
+	help
+	  This option enables Bluetooth L2CAP FCS Option
+
+config BT_L2CAP_EXT_WIN_SIZE
+	bool "Bluetooth L2CAP Extended window size Option [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	depends on BT_L2CAP_STREAM || BT_L2CAP_ENH_RET
+	help
+	  This option enables Bluetooth L2CAP Extended window size Option
+
+if BT_L2CAP_RET || BT_L2CAP_FC || BT_L2CAP_ENH_RET || BT_L2CAP_STREAM
+config BT_L2CAP_MPS
+	int "Bluetooth L2CAP MPS for retransmission and Flow control"
+	default 48
+	range 48 BT_L2CAP_TX_MTU
+	help
+	  Bluetooth L2CAP MPS for retransmission and Flow control
+
+config BT_L2CAP_MAX_WINDOW_SIZE
+	int "Maximum Windows Size of Retransmission and Flow Control"
+	default 1
+	range 1 32
+	help
+	  Maximum Windows Size of Retransmission and Flow Control.
+	  The minimum (and default) number is 1.
+	  The range is 1 to 32 for Flow Control mode and Retransmission mode.
+	  The range is 1 to 63 for Enhanced Retransmission mode.
+	  The range is 1 to 0x3FFF for Enhanced Retransmission mode with
+	  externed control.
+
+config BT_L2CAP_BR_RET_TIMEOUT
+	int "Retransmission timeout"
+	default 2000
+	range 1000 2000
+	help
+	  the Retransmission timeout shall be three times the value of flush
+	  timeout, subject to a minimum of 1 second and maximum of 2 seconds.
```

Comment:
This doesn't explain that the value is in milliseconds. In fact, when you read the help text you get the impression that the value might be in seconds. If you don't need the millisecond-level granularity, maybe just specifying in seconds is sufficient?

Reply:
Updated.